.github/workflows/pip.yml

```diff
@@ -192,10 +195,13 @@ jobs:
           CIBW_CONFIG_SETTINGS: "--global-option=egg_info --global-option=-b.dev${{ needs.pip-labels.outputs.halide_pypi_label }}"
```

Comment:
I think you can add the following here, rather than patching after the fact with `wheel`:

```suggestion
          CIBW_CONFIG_SETTINGS: "--global-option=egg_info --global-option=-b.dev${{ needs.pip-labels.outputs.halide_pypi_label }} --global-option=--build-number --global-option=${{github.run_id}}"
```

Reply:
Same idea further up.

Reply:
You might need a `--global-option=wheel` in there or to move the options around.

Reply:
Pushed a fixup now